osm-georeferencing/main.py

```diff
@@ -72,11 +71,11 @@ def map_to_annotation_event(specimen_data: Dict, results: List[Dict[str, str]],
                                                         ods_agent),
                 results))
     annotation_event = {
-        "jobId": job_id,
-        "annotations": annotations
+        'jobId': job_id,
+        'annotations': annotations
     }
     if batching:
-        annotation_event["batchMetadata"] = batch_metadata
+        annotation_event['batchMetadata'] = batch_metadata
```

Comment:
I think you can just use dot notation here :)

Reply:
sadly no because this is also adding the `batchMetadata` key to the array. No dots here

Reply:
But that works exactly the same with dots if I am not mistaken